trade-finance-manager-ui/server/controllers/utilisation-reports/add-to-an-existing-payment/index.ts

```diff
@@ -57,6 +58,7 @@ export const addToAnExistingPayment = async (req: AddToAnExistingPaymentRequest,
       availablePaymentsHeading: getAvailablePaymentsHeading(availablePaymentGroups),
       availablePaymentGroups: mapToAvailablePaymentGroupsViewModel(availablePaymentGroups),
       errors,
+      backLinkHref: getAddToExistingPaymentBackLinkToUtilisationReportReconciliationPage(reportId, feeRecordIds),
```

Comment:
Is the unit test for this passing? If so, we should update it so that it fails if something is changed/added